lib/redis.ts

```
@@ -0,0 +1,1263 @@
+  }
+
+  /**
+   * Generate a unique key for form submission
+   */
+  static generateKey(eventId: string, guestEmail: string, startTime: string): string {
+    return `${eventId}-${guestEmail}-${startTime}`.replace(/[^a-zA-Z0-9-_]/g, '_');
+  }
+}
+
+/**
+ * Types for cached data structures
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)